frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings.tsx

```diff
@@ -59,26 +65,60 @@ export const AppearanceSettings = ({
   );
   const utmTags = `?utm_source=${plan}&utm_media=static-embed-settings-appearance`;
 
-  const fontControlLabelId = useUniqueId("display-option");
   const downloadDataId = useUniqueId("download-data");
 
   return (
     <>
       <StaticEmbedSetupPaneSettingsContentSection
-        title={t`Customizing your embed’s appearance`}
+        title={t`Customizing look and feel`}
       >
-        <Text>{jt`These cosmetic options requiring changing the server code. You can play around with and preview the options here, and check out the ${(
+        <Text>{jt`These options require changing the server code. You can play around with and preview the options here. Check out the ${(
           <ExternalLink
             key="doc"
             href={`${docsUrl}${utmTags}#customizing-the-appearance-of-static-embeds`}
           >{t`documentation`}</ExternalLink>
         )} for more.`}</Text>
       </StaticEmbedSetupPaneSettingsContentSection>
       <StaticEmbedSetupPaneSettingsContentSection
-        title={t`Playing with appearance options`}
         mt="2rem"
+        data-testid="look-and-feel-settings"
       >
         <Stack spacing="1rem">
+          {canWhitelabel ? (
+            <Select
+              label={
+                <Text fw="bold" mb="0.25rem" lh="1rem">
+                  Font
+                </Text>
+              }
+              value={displayOptions.font}
+              data={[
+                {
+                  label: t`Use instance font`,
+                  // @ts-expect-error Mantine v6 and v7 both expect value to be a string
+                  value: null,
```

Comment:
@heypoom I reverted the value back to `null` as before, but added a comment explaining why I need to disable TypeScript check here instead. I think it's cleaner than trying to force it to be string as Mantine expects.